src/VideoJS.as

```diff
@@ -193,6 +194,18 @@ package{
 
         private function onAppendBufferCalled(base64str:String):void{
             var bytes:ByteArray = Base64.decode(base64str);
+            // write the bytes to the provider
+            _app.model.appendBuffer(bytes);
+        }
+
+        private function onAppendChunkReadyCalled(fnName:String):void{
+            var bytes:ByteArray;
+
+            try {
+                bytes = Base64.decode(ExternalInterface.call(fnName));
+            } catch(e:Error) {
+                bytes = Base64.decode(e.message);
```

Comment:
do we also want to remove the `ExternalInterface.marshallExceptions = true;` line ?